### PR TITLE
feat(examples): default projects and  grid search example

### PR DESCRIPTION
**Pull request by hu-po**
- Create a grid search example
- Set default project to `sweeps-examples`

### CODE REVIEW

File: examples/nested-params/using-py/train-nested-py.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ def _train_function(config):
     def train_function():
         with wandb.init(project=args.project, config=CONFIG):
             _train_function(wandb.config)
+        wandb.finish()
```
[the remark this answers is not in the document]
Comment (hu-po) [reply examples/nested-params/using-py/train-nested-py.py:76]: Fixed with https://github.com/wandb/client/pull/3899.

File: examples/nested-params/using-cli/train-nested-cli.py [hunk 2]
```diff
@@ -41,4 +41,5 @@
         wandb.log({"loss": _fake_loss})
         time.sleep(0.3)
 
+    # This line is recommended, but not required.
```
Comment (raubitsj) [inline examples/nested-params/using-cli/train-nested-cli.py:44]: not recommended

if anything you can do a context manager for wandb.init()... but this shouldnt be needed.

File: examples/nested-params/using-py/train-nested-py.py [hunk 3]
```diff
@@ -73,6 +73,8 @@ def _train_function(config):
     def train_function():
         with wandb.init(project=args.project, config=CONFIG):
             _train_function(wandb.config)
+        # This line is recommended, but not required.
+        wandb.finish()
```
Comment (raubitsj) [inline examples/nested-params/using-py/train-nested-py.py:77]: not recommended

Comment (raubitsj) [reply examples/nested-params/using-py/train-nested-py.py:77]: and never needed. as wandb.agent() calls finish.

**Review by dannygoldstein — APPROVED** [review]
lgtm!

**Pull request merged**